src/components/Button.js

```diff
@@ -2,14 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function Button(props) {
-  const { name } = props;
+  const { color, wide, name } = props;
+  const style = {};
+
+  (function isWide() {
```

Comment:
- [ ] **[OPTIONAL]** Using IIFE in react components can cause performance issues, It's also not so readable. A better approach is to use conditinal css classes, we can remove these functions and use conditional css like so:

```jsx
<button className={`${color ? '' : 'button--orange'} ${wide ? 'button--wide' : ''}`} type='button'>
```

Then, just add css classes for `button--orange` and `button--wide` with your desired css.
This is a more common pattern in applying css values conditionally.